src/default_panels/StyleTracesPanel.js

```diff
@@ -141,12 +142,12 @@ const StyleTracesPanel = ({localize: _}) => (
       <ColorPicker label={_('Border Color')} attr="marker.line.color" />
     </TraceMarkerSection>
 
-    <Section name={_('Size and Spacing')}>
-      <LayoutNumericFractionInverse label={_('Bar Width')} attr="bargap" />
-      <LayoutNumericFractionInverse label={_('Box Width')} attr="boxgap" />
-      <LayoutNumericFraction label={_('Bar Padding')} attr="bargroupgap" />
-      <LayoutNumericFraction label={_('Box Padding')} attr="boxgroupgap" />
-    </Section>
+    <LayoutSection name={_('Size and Spacing')}>
+      <NumericFractionInverse label={_('Bar Width')} attr="bargap" />
+      <NumericFractionInverse label={_('Box Width')} attr="boxgap" />
+      <NumericFraction label={_('Bar Padding')} attr="bargroupgap" />
+      <NumericFraction label={_('Box Padding')} attr="boxgroupgap" />
+    </LayoutSection>
```

Comment:
This abstraction / API is waaaay better. It looks so freaking obvious now. All the good fixes do. Made my day!